### ISSUE TITLE
[ps_emailalerts] Module v2.2.2 crashes with PrestaShop < 1.7.6

### ISSUE THREAD

**Issue by joecoolio66** [PrestaShop/PrestaShop#23743]
#### Describe the bug
Module v2.2.2 crashes with PrestaShop < 1.7.6

#### Steps to Reproduce

1. On PS 1.7.5.2, upgrade ps_emailalerts to 2.2.2
2. Attempt to make an order on FO

We upgraded this afternoon and immediately noticed that 500 errors were happening in our shop.  Customers couldn't make orders... payment was coming in but orders were not created because 500 errors were generated at validation.  

We are using prestashop 1.7.5.2

We searched through our shop and realized it was causing issues with Symfony:
```
[2021-03-17 21:13:36] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "getLocale" of class "Language". Did you mean to call "getLocaleByIso"?" at /home2/canadac3/public_html/modules/ps_emailalerts/ps_emailalerts.php line 552 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"getLocale\" of class \"Language\".\nDid you mean to call \"getLocaleByIso\"? at /home2/canadac3/public_html/modules/ps_emailalerts/ps_emailalerts.php:552)"} []
[2021-03-17 21:13:36] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
```

**We reverted back to 2.2.1 and the errors went away.**

**Comment by matks** [comment]
Bug transferred to main repository

**Comment by matks** [comment]
Will be fixed by release 2.3.0 of module ps_emailalerts (PR https://github.com/PrestaShop/ps_emailalerts/pull/81)

**Comment by aDamelincourt** [comment]
Hello @matks 
It's ok on PS1.7.7.3 for ps_emailalerts
But on PS1.7.5.2, I can't finalize my cart, after payment choice, I get a 500 error, in BO, I have got order, but it's empty
